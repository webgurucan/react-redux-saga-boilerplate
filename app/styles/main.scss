// *** Vendor
@import '~sass-mediaqueries/media-queries';

// *** Base
@import 'variables/common';
@import 'variables/bootstrap';
@import 'variables/typography';
@import 'modules/icons';

// *** Utilities
@import 'utilities/functions';
@import 'utilities/mixins';
@import 'utilities/grid';

// *** Vendor
@import 'vendor/bootstrap';

// *** Modules
@import 'modules/animations';
@import 'modules/buttons';
@import 'modules/transitions';

// *** Layout
@import 'layout/global';
@import 'layout/app';

// *** Components
@import 'components/alert';
@import 'components/footer';
@import 'components/header';
@import 'components/loader';
@import 'components/logo';
@import 'components/system-alerts';

// *** Containers
@import 'containers/github';

// *** Routes
@import 'routes/home';
@import 'routes/private';
@import 'routes/not-found';
