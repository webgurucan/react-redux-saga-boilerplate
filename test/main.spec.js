import main from '../app/scripts/main';

describe('main', () => {
  it('should match the snapshot', () => {
    expect(String(main)).toMatchSnapshot();
  });
});
