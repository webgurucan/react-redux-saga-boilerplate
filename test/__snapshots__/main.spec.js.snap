exports[`main should match the snapshot 1`] = `
"function renderApp(RootComponent){/* istanbul ignore next */++__cov_eTax9T2rmusaTjiZjUgmZdbZEXE.f[\'1\'];var target=/* istanbul ignore next */(++__cov_eTax9T2rmusaTjiZjUgmZdbZEXE.s[\'1\'],document.getElementById(\'react\'));

/* istanbul ignore if *//* istanbul ignore next */++__cov_eTax9T2rmusaTjiZjUgmZdbZEXE.s[\'2\'];
if(target){
/* istanbul ignore next */_reactDom2.default.render(
/* istanbul ignore next */_react2.default.createElement(/* istanbul ignore next */_reactHotLoader.AppContainer,/* istanbul ignore next */null,
/* istanbul ignore next */_react2.default.createElement(RootComponent,/* istanbul ignore next */{store:/* istanbul ignore next */_store2.default,history:/* istanbul ignore next */(0,_reactRouterRedux.syncHistoryWithStore)(/* istanbul ignore next */_reactRouter.browserHistory,/* istanbul ignore next */_store2.default)})),

target);

}else/* istanbul ignore next */{++__cov_eTax9T2rmusaTjiZjUgmZdbZEXE.b[\'1\'][0];}
}"
`;
